rep-I0001.rst

```diff
@@ -109,9 +108,11 @@ The dynamic joint state is meant to mimic both the ROS JointState and FollowJoin
         accelerations[] 
         effort[]
         position_desired[]
-        position_errors[]
+        position_error[]
         velocity_desired[]
-        velocity_errors[]
+        velocity_error[]
+        accel_desired[]
+        accel_error[]
```

Comment:
Why revert to singular only here? To get this more in-line with the [JointState](http://docs.ros.org/api/sensor_msgs/html/msg/JointState.html) msg, I'd make all field names singular (ie: `position`, `velocity`, `acceleration`, etc).